firestore/src/include/firebase/firestore.h

```diff
@@ -405,6 +406,41 @@ class Firestore {
       EventListener<void>* listener);
 #endif  // !defined(FIREBASE_USE_STD_FUNCTION) || defined(DOXYGEN)
 
+  /**
+   * Loads a Firestore bundle into the local cache.
+   *
+   * @param bundle A string representing the bundle to be loaded.
+   * @return A `Future` that is resolved when the loading is either completed,
+   * or aborted due to an error.
+   */
+  virtual Future<LoadBundleTaskProgress> LoadBundle(const std::string& bundle);
+
+  /**
+   * Loads a Firestore bundle into the local cache, with provided callbacks
+   * executed for progress updates.
+   *
+   * @param bundle A string representing the bundle to be loaded.
```

Comment:
Yeah, in Android things are clearer since the objects passed to loadBundle obviously ARE the bundle, cast in some data structure. In C++, does this string (a) "identify" the bundle to be loaded, or (b) actually contain the bundle data? If (a) then I'd say "A string identifying the bundle...". If (b), then yes it may be a little clear to say "A string containing the bundle...."

Reply:
Ping.

Reply:
Done.